<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ú–µ–Ω—é</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(31, 3, 54);
background-color: rgb(212, 229, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonSolveUrav">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>–†–µ—à–∏—Ç—å  —Å–∏—Å—Ç–µ–º—É —É—Ä–∞–≤–Ω–µ–Ω–∏–π!</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>801</width>
      <height>448</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>85</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-style:italic; color:#120b5d;&quot;&gt;–†–µ—à–µ–Ω–∏–µ —Å–∏—Å—Ç–µ–º—ã –ª–∏–Ω–µ–π–Ω—ã—Ö —É—Ä–∞–≤–Ω–µ–Ω–∏–π –ø–µ—Ä–≤–æ–≥–æ –ø–æ—Ä—è–¥–∫–∞ –º–µ—Ç–æ–¥–æ–º –ì–∞—É—Åc–∞.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(226, 242, 255);</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;h2 style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;‚úÖ &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline;&quot;&gt;–ü–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–µ —Å–∏–º–≤–æ–ª—ã –∏ —Ñ–æ—Ä–º–∞—Ç —É—Ä–∞–≤–Ω–µ–Ω–∏–π&lt;/span&gt;&lt;/h2&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;üìå –ú–æ–∂–Ω–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞—Ç—å:&lt;/span&gt;&lt;/h3&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;üî§ &lt;span style=&quot; font-weight:700;&quot;&gt;–õ–∞—Ç–∏–Ω—Å–∫–∏–µ –±—É–∫–≤—ã&lt;/span&gt; –¥–ª—è –ø–µ—Ä–µ–º–µ–Ω–Ω—ã—Ö&lt;br /&gt;(–Ω–∞–ø—Ä–∏–º–µ—Ä: &lt;span style=&quot; font-family:'Courier New';&quot;&gt;x&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;y&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;a&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;b1&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;z2&lt;/span&gt;)&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;üî¢ &lt;span style=&quot; font-weight:700;&quot;&gt;–ß–∏—Å–ª–∞&lt;/span&gt;, –≤–∫–ª—é—á–∞—è –¥–µ—Å—è—Ç–∏—á–Ω—ã–µ&lt;br /&gt;(–Ω–∞–ø—Ä–∏–º–µ—Ä: &lt;span style=&quot; font-family:'Courier New';&quot;&gt;2&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;3.5&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;-0.25&lt;/span&gt;)&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ûï‚ûñ &lt;span style=&quot; font-weight:700;&quot;&gt;–û–ø–µ—Ä–∞—Ü–∏–∏:&lt;/span&gt; &lt;span style=&quot; font-family:'Courier New';&quot;&gt;+&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;-&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;=&lt;/span&gt;&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ê£ &lt;span style=&quot; font-weight:700;&quot;&gt;–ü—Ä–æ–±–µ–ª—ã&lt;/span&gt;&lt;br /&gt;(–∏–≥–Ω–æ—Ä–∏—Ä—É—é—Ç—Å—è –ø—Ä–∏ –æ–±—Ä–∞–±–æ—Ç–∫–µ)&lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;üìê –§–æ—Ä–º–∞—Ç —É—Ä–∞–≤–Ω–µ–Ω–∏—è:&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;–ö–∞–∂–¥–æ–µ —É—Ä–∞–≤–Ω–µ–Ω–∏–µ –¥–æ–ª–∂–Ω–æ –±—ã—Ç—å &lt;span style=&quot; font-weight:700;&quot;&gt;–ª–∏–Ω–µ–π–Ω—ã–º&lt;/span&gt;.&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;üîç &lt;span style=&quot; font-weight:700;&quot;&gt;–ü—Ä–∏–º–µ—Ä –¥–æ–ø—É—Å—Ç–∏–º–æ–≥–æ —É—Ä–∞–≤–Ω–µ–Ω–∏—è:&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:1; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;2x + 3y = 5&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;‚ö†Ô∏è –ù–µ –ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ—Ç—Å—è:&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ùå –£–º–Ω–æ–∂–µ–Ω–∏–µ —Å —Å–∏–º–≤–æ–ª–æ–º &lt;span style=&quot; font-family:'Courier New';&quot;&gt;*&lt;/span&gt; (–Ω–∞–ø—Ä–∏–º–µ—Ä: &lt;span style=&quot; font-family:'Courier New';&quot;&gt;2*x&lt;/span&gt;)&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ùå –°–∫–æ–±–∫–∏ &lt;span style=&quot; font-family:'Courier New';&quot;&gt;( )&lt;/span&gt;&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ùå –°—Ç–µ–ø–µ–Ω–∏ &lt;span style=&quot; font-family:'Courier New';&quot;&gt;^&lt;/span&gt;&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ùå –î–µ–ª–µ–Ω–∏–µ &lt;span style=&quot; font-family:'Courier New';&quot;&gt;/&lt;/span&gt;&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ùå –†—É—Å—Å–∫–∏–µ –±—É–∫–≤—ã (–Ω–∞–ø—Ä–∏–º–µ—Ä: &lt;span style=&quot; font-family:'Courier New';&quot;&gt;–∞&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;–±&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;—Ö&lt;/span&gt;)&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;‚ùå –°–ø–µ—Ü–∏–∞–ª—å–Ω—ã–µ —Å–∏–º–≤–æ–ª—ã &lt;span style=&quot; font-family:'Courier New';&quot;&gt;@&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;#&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;!&lt;/span&gt;, –∏ —Ç.–ø.&lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;üß† –í–∞–∂–Ω–æ –ø–æ–º–Ω–∏—Ç—å:&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;–¢–æ–ª—å–∫–æ &lt;span style=&quot; font-weight:700;&quot;&gt;–æ–¥–∏–Ω –∑–Ω–∞–∫ &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-weight:700;&quot;&gt;=&lt;/span&gt; –≤ –∫–∞–∂–¥–æ–º —É—Ä–∞–≤–Ω–µ–Ω–∏–∏.&lt;/li&gt;
&lt;li style=&quot; font-size:10pt; font-style:italic;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;–ò–º—è –ø–µ—Ä–µ–º–µ–Ω–Ω–æ–π –º–æ–∂–µ—Ç —Å–æ–¥–µ—Ä–∂–∞—Ç—å &lt;span style=&quot; font-weight:700;&quot;&gt;–ª–∞—Ç–∏–Ω—Å–∫–∏–µ –±—É–∫–≤—ã –∏ —Ü–∏—Ñ—Ä—ã&lt;/span&gt;:&lt;br /&gt;‚úÖ &lt;span style=&quot; font-family:'Courier New';&quot;&gt;a1&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;b2&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;x&lt;/span&gt;&lt;br /&gt;‚ùå &lt;span style=&quot; font-family:'Courier New';&quot;&gt;–∞&lt;/span&gt; (—Ä—É—Å—Å–∫–∞—è &amp;quot;–∞&amp;quot;), &lt;span style=&quot; font-family:'Courier New';&quot;&gt;x@&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;x^2&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelVvod">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(196, 209, 255);</string>
       </property>
       <property name="text">
        <string>–í–≤–µ–¥–∏—Ç–µ —Å–∏—Å—Ç–µ–º—É —É—Ä–∞–≤–Ω–µ–Ω–∏–π (–¥–æ–ø—É—Å—Ç–∏–º–æ –º–∞–∫—Å–∏–º—É–º 5 —É—Ä–∞–≤–Ω–µ–Ω–∏–π –≤ —Å–∏—Å—Ç–µ–º–µ):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelUrav1">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;–£—Ä–∞–≤–Ω–µ–Ω–∏–µ ‚Ññ1:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditUrav1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelUrav2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;–£—Ä–∞–≤–Ω–µ–Ω–∏–µ ‚Ññ2:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditUrav2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelUrav3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;–£—Ä–∞–≤–Ω–µ–Ω–∏–µ ‚Ññ3:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditUrav3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelUrav4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;–£—Ä–∞–≤–Ω–µ–Ω–∏–µ ‚Ññ4:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditUrav4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelUrav5">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;–£—Ä–∞–≤–Ω–µ–Ω–∏–µ ‚Ññ5:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditUrav5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonDelUrav_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>341</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>-–£–±—Ä–∞—Ç—å —É—Ä–∞–≤–Ω–µ–Ω–∏–µ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddUrav">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>371</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>+–î–æ–±–∞–≤–∏—Ç—å —É—Ä–∞–≤–Ω–µ–Ω–∏–µ</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
